api/turing/cmd/main.go

```diff
@@ -107,6 +108,12 @@ func main() {
 		defer sentry.Close()
 	}
 
+	// Run batch runners
+	go batchrunner.RunBatchRunners(
```

Comment:
Is it required to do some cleanup on app termination?

Reply:
Not strictly required as the job is able to repickup itself via the updateStatus method (https://github.com/ashwinath/turing/blob/batch-ensembler-orchestration/api/turing/batch/runner/ensembling.go#L53)